frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -21,6 +21,7 @@ export interface ITableQueryData {
   sortHeader: string;
   sortDirection: string;
   showInheritedTable?: boolean; // Only used for policies tables
+  inheritedTable?: boolean; // Only used for policies tables
```

Comment:
@RachelElysia are both `showInheritedTable` and `inheritedTable` necessary?

Reply:
Yeah, unfortunately.`showInheritedTable` for whether the inherited table is showing or not showing and `inheritedTable` is for which policy table is being edited so if `inheritedTable` is set we're changing the sortdirection/sortheader/page for inherited table and not the top table.

Reply:
Ah, gotcha. Maybe for future work a clearer name could be something like `editingInheritedTable` maybe? Nbd though, nice job.

Reply:
Somehow this line was removed from the final commit, which broke the `make-generate` build step.